script/generate_incremental_table

```diff
@@ -78,8 +78,8 @@ def query(day, entrypoint, destination_table, query_options, query_file):
             "--replace",
             "--destination_table=" + destination_table + day.strftime("$%Y%m%d"),
             "--parameter=submission_date:DATE:" + day.isoformat(),
-            "--time_partitioning_field=submission_date",
```

Comment:
If an existing table is clustered, either neither or both of --time_partitioning_field and --clustering_fields must be specified, or neither.